bundles/org.openhab.binding.boschshc/src/main/java/org/openhab/binding/boschshc/internal/devices/intrusion/IntrusionDetectionHandler.java

```diff
@@ -122,18 +122,18 @@ public void handleCommand(ChannelUID channelUID, Command command) {
 
         switch (channelUID.getId()) {
             case CHANNEL_ARM_ACTION:
-                if (command instanceof StringType) {
-                    armIntrusionDetectionSystem((StringType) command);
+                if (command instanceof StringType action) {
```

Comment:
variable name should be `stringCommand` (or if we want it to be even more specific: `profileId` or `profileIdCommand`)